src/resources/single/__init__.py

```diff
@@ -0,0 +1 @@
+from.Review import SingleReview
```

Comment:
This is good here. Remove the other one in models. Don't forget to also export your SingleReview class from `resources/__init__.py`.